incuna-sass/components/_typography.sass

```diff
@@ -0,0 +1,216 @@
+// Based on Simple Typography by Adam Marsden and contributors. 
+// Licensed under the MIT license.
+// https://github.com/AdamMarsden/simple-typography
+
+// Font Family
+$base-font-family: 'Lato' !default
+
+// Default font sizes
+$base-font-size: 16px !default
+
+$paragraph-size: 15px !default
+
+$heading-size: 30px !default
+
+// Make type normal across browsers
+*,
+html,
+body
+    -webkit-text-size-adjust: 100%
+    -ms-text-size-adjust: 100%
+
+html,
+body
+    @include font($base-font-family)
+    font:
+        // family: $base-font-family
+        size: $base-font-size
+
+// Headings
+h1,
+h2,
+h3,
+h4,
+h5,
+h6
+    margin:
+        top: 0.5em
+        right: 0
+        bottom: 0.75em
+        left: 0
+    line-height: 1.25em
+
+h1
+    font:
+        size: $heading-size
+
+h2
+    font:
+        size: $heading-size / 1.3
+
+h3
+    font:
+        size: $heading-size / 1.5
+
+h4
+    font:
+        size: $heading-size / 1.7
+
+h5
+    font:
+        size: $heading-size / 1.8
+
+h6
+    font:
+        size: $heading-size / 2
+
+// Paragraphs
+p
+    margin: 0
+        bottom: 1.25em
```

Comment:
yeah, possibly, what would you want to call it? `$paragraph-spacing`? I can never decide if `spacing` is better than `padding` or `margin`